runtime/oti/j9port.h

```diff
@@ -508,6 +508,11 @@ typedef struct J9ProcessorDesc {
 /* STFLE bit 57 - Message-security-assist-extension-5 facility */
 #define J9PORT_S390_FEATURE_MSA_EXTENSION_5 57
 
+/* zLinux features
+ * Auxiliary Vector Hardware Capability (AT_HWCAP) features for zLinux.
+ */
+#define J9PORT_HWCAP_S390_GS 16384
```

Comment:
Could we use a hex 0x4000 for this value instead? It is a lot more clear IMO. Also "zLinux" is a deprecated term. We should try to use "Linux on Z" instead.

Reply:
fixed the comment and changed the value to hex.